astropy/coordinates/transformations/graph.py

```diff
@@ -514,10 +515,11 @@ def to_dot_graph(
         lines.append("}")
         dotgraph = "\n".join(lines)
 
+        if isinstance(savefn, str):
+            savefn = Path(savefn)
         if savefn is not None:
```

Comment:
I would dedent the following code block
```python
if savefn is None:
    return dotgraph
if isinstance(savefn, str):
...
```
But that would make the patch seem quite large. The minimal acceptable change for this pull request would be 
```suggestion
        if savefn is not None:
            if isinstance(savefn, str):
                savefn = Path(savefn)
```

Reply:
done